templates/downloads.html.ep

```diff
@@ -38,7 +38,13 @@
       % }
       </p>
       <a class="btn btn-dark kick-b" href="<%= url_for 'downloads-rakudo' %>">All releases</a>
+      
+      <h5>Use a package manager to install the binary release</h5>
 
+      <p>Install Rakudo-on-MoarVM with scoop:
```

Comment:
Shouldn't it be "Scoop" with a capital "S"? That's how it is written on its [homepage](https://github.com/lukesampson/scoop/wiki). Same in line 59 (you didn't touch that line, but should be corrected anyways).